### PR TITLE
moved p_log_posterior() as argument to bioassay_posterior_density_plot

### PR DESCRIPTION
**Pull request by topipa**
Previously the function assumed that there exists a `p_log_posterior()` in the global environment.

### CODE REVIEW

**Comment by topipa** [comment]
Good point. Done!

**Comment by MansMeg** [comment]
A short question, could you check if your PR would create merge conflicts with my PR?

File: rpackage/DESCRIPTION [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Package: aaltobda
 Type: Package
 Title: Functionality and Data for the Aalto Course on Bayesian Data Analysis
-Version: 0.3
+Version: 0.31
```
Comment (MansMeg) [inline rpackage/DESCRIPTION]: I would use 0.3.1 instead.

**Comment by MansMeg** [comment]
I had a small comment on the version number, but otherwise. Good work!

**Comment by topipa** [comment]
> A short question, could you check if your PR would create merge conflicts with my PR?

It does actually, at least for ex4.pdf. I can fix the conflicts once your PR is merged.

**Comment by MansMeg** [comment]
Great! Let's wait for @avehtari to review PR #48 , and when that is good to go, so is this one. Great work @topipa !

**Comment by MansMeg** [comment]
Alright! Now the merge conflict needs to be solved. Then we can merge it.

**Comment by topipa** [comment]
Merge conflict solved. Can be merged!

**Pull request merged**